andela/bp-esa-frontend
# #167485260 Redirect to partner profile when partner name is clicked in card view

#### What does this PR do?

This PR adds functionality to Redirect to partner profile when a partner name is clicked in card view

#### Description of Task to be completed?

* Redirect to partner profile when a partner name is clicked in Card view.
* Add link icon to inform users of the availability of link on the partner name.

#### How should this be manually tested?
1.Git clone the repository.
2.Checkout to the ft-add-partner-link-in-card-view-#167485260 branch.
3. Start the server npm run start (or the yarn equivalent).
4.Clone the backend repository, https://github.com/andela/bp-esa-backend.
5. Checkout to the backend develop branch.
6. Start the backend server npm run start:dev (or the yarn equivalent).
7.Login to the ESA application and click the card view icon.
8. Click the partner's name/link icon beside the partner name. You should be redirected to the partner's profile page.

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?
[#167485260](https://www.pivotaltracker.com/story/show/167485260)

#### Screenshots (If applicable)

N/A
